app/Models/Tag.php

```diff
@@ -30,7 +30,7 @@ class Tag extends Model
 
     protected $guarded = [];
 
-    public function tag_relations(){
+    public function tag_relations() {
```

Comment:
__[code_sniffer]__
* Method name "Tag::tag_relations" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->